pkg/volume/secret/secret_test.go

```diff
@@ -124,8 +123,7 @@ func TestPlugin(t *testing.T) {
 	doTestSecretDataInVolume(volumePath, secret, t)
 
 	metrics, err := builder.GetMetrics()
-	assert.NotEmpty(t, metrics)
-	assert.NoError(t, err)
+	volume.AssertMetricsOk(t, metrics, err)
```

Comment:
I am more concerned about stuff that would be added to the test in the future that could get skipped.  Don't feel too strongly about this since it is only on non-linux.